package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@mdn/yari",
   "version": "0.1.6",
   "scripts": {
-    "build": "cross-env NODE_ENV=production node build/cli.js",
+    "build": "(test -f ssr/dist/main.js || yarn build:ssr) && cross-env NODE_ENV=production node build/cli.js",
```

Comment:
This is not all. You'd also need the client build too. That's why you have to first run `yarn prepare-build` before you can run `yarn build`.

Reply:
Thanks for prompt review!

So is this the desired result?
```
"build": "(test -f client/build/index.html || yarn build:client) && (test -f ssr/dist/main.js || yarn build:ssr) && cross-env NODE_ENV=production node build/cli.js"
```

Or is it better to just do this?
```
"build": "yarn prepare-build && cross-env NODE_ENV=production node build/cli.js"
```
Frankly, I don't like that in this version `yarn build` calls `yarn prepare-build` which calls `yarn build --spas-only`. Seems a bit circular...

Reply:
Well, I tested that with a fresh repo and it works either way.

Specifically, I did `git clone ...`, then `yarn` to install deps, then `yarn build` and got docs building.